server/app.js

```diff
@@ -10,6 +10,7 @@ app.get('/foo', (req, res) => res.send('bar'));
 app.get('/bar', (req, res) => res.send('foo'));
 
 app.use(express.static(path.join(__dirname, '/public')));
+app.use(express.static(path.join(__dirname, '/../app/dist')));
```

Comment:
Convert the string with path separators here and above to just be multiple arguments  to `path.join`.